torch_glow/tests/models/resnet_test.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torchvision
+
+import unittest
+import torch_glow
+
+from tests.utils import jitVsGlow
+
+
+class TestResnet(unittest.TestCase):
+    def test_quantized_resnet18(self):
+        """Basic test of the PyTorch quantized resnet on Glow."""
+
+        torch._C._jit_set_profiling_executor(False)
+        torch._C._jit_set_profiling_mode(False)
+
+        # model could be:
+        # torchvision.models.quantization.resnet18
+        # torchvision.models.quantization.resnet50
+        # torchvision.models.quantization.resnext101_32x8d
+        model = torchvision.models.quantization.resnet18(pretrained=True,
+                                                         quantize=True)
+
+        model.qconfig = torch.quantization.get_default_qconfig('fbgemm')
+
+        # x could be generated by tests.utils.get_image(),
+        # by torchvision.dataset or by totally random torch.randn(1,3,224,224).
+        # For stability and efficiency of unit test,
+        # we use determined random generated inputs
+        torch.manual_seed(0)
+        x = torch.randn(1, 3, 224, 224)
+
+        torch_answer = model(x).topk(3)
+
+        with torch.no_grad():
+            torch_glow.enableFusionPass()
+
+            traced_model = torch.jit.trace(model, x)
+            traced_model.eval()
+
+            glow_answer = traced_model(x).topk(3)
```

Comment:
Great to see this end to end example, a couple of comments about the glow API:
The current API is non-intuitive. It is not clear why enabling fusion pass in torch_glow affects the traced model.
It will be more intuitive to do something like:
```python
traced_model = torch.jit.trace(model,x)
glow_model = torch_glow.convert(model, glow_options)
torch_answer = traced_model(x)
glow_answer = glow_model(x)
```
Is this possible? (cc: @yinghai , @jackm321 )

Reply:
Maybe but python interface isn't really what we focus on. This is for convenience of testing.